src/control-plane/backend/stack-action-state-machine-construct.ts

```diff
@@ -165,6 +165,7 @@ export class StackActionStateMachine extends Construct {
             'globalaccelerator:*',
             'kms:*',
             'athena:*',
+            'servicecatalog:*',
```

Comment:
Let's grant the least privileges to the service catalog.

Reply:
done